### ISSUE TITLE
case sensitivity issue

### ISSUE THREAD

**Issue by Alex-Jordan** [openwebwork/webwork-open-problem-library#692]
On a new Mac, I just cloned the OPL. At the very end, I get this warning:

```
...
Checking out files: 100% (70316/70316), done.
warning: the following paths have collided (e.g. case-sensitive paths
on a case-insensitive filesystem) and only one from the same
colliding group is in the working tree:

  'Contrib/CUNY/CityTech/CollegeAlgebra_Trig/ReducingRationalExpressions/monoDenom-Scaff.pg'
  'Contrib/CUNY/CityTech/CollegeAlgebra_Trig/ReducingRationalExpressions/monoDenom-scaff.pg'
```

It leaves with this file as "modified" but I can't `checkout` or stage and commit the file. 

At some point, some commit was made using lowercase "s" in "scaff" and at some other point, with capital "S". I think I could figure out how to deal with this on my system, but I'm not sure if this is an issue for the repo in general and people cloning the OPL or updating it. Does anyone understand this well enough to know what to do?

**Comment by jwj61** [comment]
I think it is an issue for people with a mac.  On unix/linux distributions, it should be fine.

@drdrew42 :
If one of them should be deleted, we should do that.  Otherwise, it would be good to rename one of them so that the filenames differ by more than upper/lower case.

**Comment by Alex-Jordan** [comment]
Will deleting or renaming will fix everything? When someone does an update, will git try to fastforward through the brief period when there were two of them, and still get stuck, not yet aware of a later commit that resolves the issue?

I renamed one of them, staged and committed, and that let me complete what I needed to do.

Apparently MacOS and Windows are case-insensitive. Seems like it would be rare for this to affect a production server. But for example it would hinder local editing on your Mac/Windows system to commit some typo fix or whatever.
